controller/space_iterations_test.go

```diff
@@ -131,8 +135,11 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIterationWithOptionalValues
 	ci := createSpaceIteration(iterationName, &iterationDesc)
 	application.Transactional(rest.db, func(app application.Application) error {
 		repo := app.Spaces()
+		spaceTemplate, err := spacetemplate.CreateEmptySpaceTemplateWithRepo(context.Background(), app.SpaceTemplates())
+		require.Nil(rest.T(), err)
 		testSpace := space.Space{
-			Name: "TestSuccessCreateIterationWithOptionalValues-" + uuid.NewV4().String(),
+			Name:            "TestSuccessCreateIterationWithOptionalValues-" + uuid.NewV4().String(),
```

Comment:
I'm not using it in hopefully all places. See cb9fa478ab4d4b7d0db6c70e8510065cd3b888cd. But even though I might have changed the lines above, I only changed the indention.